spec/models/note_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+RSpec.describe Note, type: :model do
+  include ActiveSupport::Testing::TimeHelpers
+  let(:author){Fabricate(:user, email: "admin@art.sy")}
+  let(:submission){Fabricate(:submission)}
```

Comment:
It seems like this file should be [prettified](https://github.com/prettier/plugin-ruby).

Reply:
yeah, i just realized this was available. for some reason the format option in vscode (which points to rubocop) did not seem to be working.